client/app/components/guides/guides.html

```diff
@@ -1,15 +1,13 @@
-<div class="oui-tile mb-5" data-ng-if="!!guideConfiguration && guideConfiguration.length > 0">
+<div class="oui-tile oui-tile_collapsible mb-5" data-ng-if="!!guideConfiguration && guideConfiguration.length > 0">
     <h3 class="oui-tile__title" data-ng-bind="guidesTitle"></h3>
     <div class="oui-tile__body">
-        <div class="oui-tile__item" data-ng-repeat="guideSection in guideConfiguration track by $index">
-            <div class="oui-tile__item_button">
-                <button type="button" class="oui-tile__button oui-button oui-button_link oui-button_full-width d-flex text-wrap" data-ng-click="toggleGuides($index)"
-                        data-ng-class="{ 'showGuidesStatus': item.active }">
-                    <span data-ng-bind="tr(guideSection.title)"></span>
-                    <span data-ng-show="showGuidesStatus[$index]" class="fa fa-chevron-up my-auto ml-auto"></span>
-                    <span data-ng-show="!showGuidesStatus[$index]" class="fa fa-chevron-down my-auto ml-auto"></span>
-                </button>
-            </div>
+        <div class="oui-tile__item oui-tile__item_button" data-ng-repeat="guideSection in guideConfiguration track by $index">
+            <button type="button" class="oui-tile__button oui-button oui-button_link oui-button_full-width d-flex text-wrap" data-ng-click="toggleGuides($index)"
+                    data-ng-class="{ 'showGuidesStatus': item.active }">
```

Comment:
Class `showGuidesStatus` is no longer used, so I think you can get rid of `data-ng-class`.